src/ui/RequestBuilder.cpp

```diff
@@ -332,6 +332,7 @@ RequestBuilder::RequestBuilder(url_source_request_data *request_data,
 	outputTypeComboBox->addItem("Text");
 	outputTypeComboBox->addItem("JSON");
 	outputTypeComboBox->addItem("XML");
```

Comment:
while you're here - why not add `(XPath)` to this sessting

Reply:
OK, just added.